src/foam/dao/NullDAO.js

```diff
@@ -22,6 +22,18 @@ foam.CLASS({
 
   documentation: 'A Null pattern (do-nothing) DAO implementation.',
 
+  properties: [
+    {
+      name: 'of',
+      factory: function() {
+        return this.cls_;
```

Comment:
Isn't this going to return NullDAO? Is that useful? Better to set value: foam.core.FObject?

Reply:
Had to pick something.  Need to satisfy the getOf().getId() in PMInfoDAO. 
I'll update.

Reply:
Does AbstractFObject work?